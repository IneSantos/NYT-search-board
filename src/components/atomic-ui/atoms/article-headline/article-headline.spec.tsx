import React from "react";
import { render } from "@testing-library/react";
import ArticleHeadline from "./article-headline";

describe("ArticleHeadline atom tests", () => {
  test(`ArticleHeadline component render`, () => {
    const { container } = render(
      <ArticleHeadline headline="This is a headline" />
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
