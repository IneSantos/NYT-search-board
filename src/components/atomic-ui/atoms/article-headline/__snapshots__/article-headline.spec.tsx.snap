// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleHeadline atom tests ArticleHeadline component render 1`] = `
<h3
  class="title"
>
  This is a headline
</h3>
`;
